src/app/views/media-coverage/media-coverage.component.scss

```diff
@@ -35,9 +35,9 @@ hr{
 div.d-flex{
   max-width: 100%;
 }
-*{
-  overflow-x: hidden;
-}
+// *{
+//   overflow-x: hidden;
```

Comment:
Please explain this

Reply:
Atually I took the code for this page from Workshop and Meeting Components, there also Overflow-x was there, so I didn't knew that it was creating problem, but once I found this was the only reason, I fixed it across. I commented out everywhere, just in case we need it later so. Once All will settle, will refine CSS once and will remove Commented Codes from everywhere.

Reply:
Okay, but from next time onwards please do not mark conversation as resolved by your own